autoload/ensime.vim.py

```diff
@@ -9,10 +9,18 @@
 
 def ensime_init_path():
     path = os.path.abspath(inspect.getfile(inspect.currentframe()))
-    if path.endswith('/rplugin/python/ensime.py'):  # nvim rplugin
```

Comment:
I see your point. normath is doing too many things so I shortened the patch using join